sdcm/cluster.py

```diff
@@ -2061,17 +2067,21 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo, auth_token, segments_per_r
         self.log.debug("Copying TLS files from data_dir to node")
         self.remoter.send_files(src='./data_dir/ssl_conf', dst='/tmp/')  # pylint: disable=not-callable
 
-        self.download_scylla_manager_repo(scylla_mgmt_repo)
+        if package_url:
+            package_names = '{0}scylla-manager-server* {0}scylla-manager-client*'.format(package_url)
```

Comment:
Maybe I'm being a bit too paranoid, but I feel like it's possible that someone will use a url without a `/` at the end and the joined urls will break
We can use os.path.join to join them?

Reply:
You are correct that path are better join with the right tools 

Pathlib or os.path.join should be the right thing to do

Reply:
it was correct, but the function it is calling to set this value adds the trailing '/' if it doesn't have:
```
    if not tmp_dir.endswith('/'):
        tmp_dir += '/'
```

Reply:
Yeah, I can see that now

Reply:
you are correct that in this case it won't be a problem, but we still should strive to use the right tool for paths, next time there won't be a code like that preventing the issue.

Reply:
i would, if it wasn't already enforced by the code i'm calling.